pandas/io/parsers.py

```diff
@@ -1620,6 +1620,12 @@ def __init__(self, src, **kwds):
 
         if self.usecols:
             usecols = _evaluate_usecols(self.usecols, self.orig_names)
+
+            #gh-14671
```

Comment:
k that's fine, maybe think about making this an actual ``_validate_usecols`` (if it reduces code and is more clear).